csfieldguide/chapters/content/en/coding-encryption/sections/the-key-distribution-problem.md

```diff
@@ -70,13 +72,13 @@ Some of the language may not be suitable for use in class, so discretion is need
 
 ## Public key systems
 
-One of the remarkable discoveries in computer science in the 1970s was a method called *public key encryption*, where it's fine to tell everyone what the key is to encrypt any messages, but you need a special private key to decrypt it.
+One of the remarkable discoveries in computer science in the 1970s was a method called {glossary-link term="public-key-cryptography"}public key encryption{glossary-link end}, where it's fine to tell everyone what the key is to encrypt any messages, but you need a special private key to decrypt it.
```

Comment:
I don't think this needs a glossary tag here. The following sentence pretty much explains what it is, and the glossary link talks about public key cryptography in general, not just public key encryption.

Reply:
They're the same thing: "public key" is important here, the rest is semantic. Cryptography is the Wikipedia version, encryption is used for the same thing in several other places (including here) so I could change a word to be consistent?

Also putting links in the sentence where it's explained is both consistent with what was done before I started, and appears to fit the [docs](https://cs-field-guide.readthedocs.io/en/latest/author/content_style_guide.html#content) (though ambiguous)

> Any jargon used needs to be clearly explained and should be considered as a glossary definition.

Reply:
My main issue is that I think glossary definitions should be as concise as possible, and this one (in this particular context) is quite long. It feels like the information I get from the glossary definition is unecessarily interruptive seeing as the next sentence defines it just fine.

I think 'putting links in the sentence where it's explained' should be considered case by case.